dmtestutils/api_model_stubs/base.py

```diff
@@ -27,13 +30,55 @@ def test_a_view_that_needs_briefs_from_the_api(mocked_get_brief_method):
 
     def __init__(self, **kwargs):
         self.response_data = self.default_data.copy()
+        self._normalise_kwargs(kwargs)
+        self.response_data.update(**kwargs)
+
+    def _normalise_kwargs(self, kwargs):
+        """Turn any snake_case kwargs into camelCase
+
+        Modifies the dictionary :kwargs: in-place.
 
+        Requires a list of tuples at self.optional_keys to
+        map between kwargs and keys in self.response_data.
+        """
         # Backwards compatibility for snake case kwargs
         for camelcase_key, snakecase_kwarg in self.optional_keys:
             if kwargs.get(snakecase_kwarg) is not None:
-                self.response_data[camelcase_key] = kwargs.pop(snakecase_kwarg)
+                kwargs[camelcase_key] = kwargs.pop(snakecase_kwarg)
 
-        self.response_data.update(**kwargs)
+    def _format_framework(self, slug, oldstyle=False):
+        """Return a dictionary with correct keys for framework slug"""
+        family, iteration = re.match(r"^(?P<family>[a-z-]*)(?:-(?P<iteration>\d+))?$", slug).groups()
+        name = (
+            {
+                "g-cloud": "G-Cloud",
+                "digital-outcomes-and-specialists": "Digital Outcomes and Specialists",
+            }.get(family,
+                  slug.translate(str.maketrans("-", " ")).title())
```

Comment:
Any benefit to `translate` over `replace`?

Reply:
Nope, I can't believe I forgot about replace :man_facepalming: